mass_editing/models/mass_editing.py

```diff
@@ -14,3 +14,13 @@ class MassEditing(models.Model):
     line_ids = fields.One2many(
         comodel_name="mass.editing.line", inverse_name="mass_editing_id"
     )
+    apply_domain_in_lines = fields.Boolean(
+        string="Apply domain in lines",
+        compute="_compute_apply_domain_in_lines"
+    )
+
+    @api.depends("line_ids")
+    def _compute_apply_domain_in_lines(self):
+        self.apply_domain_in_lines = False
+        if self.line_ids.filtered(lambda x: x.apply_domain):
+            self.apply_domain_in_lines = True
```

Comment:
Not multi-record aware:
```suggestion
        for record in self:
            record.apply_domain_in_lines = any(record.line_ids.mapped("apply_domain"))
```